packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -483,6 +485,11 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
 
         drawStaticOval(canvasEl!, videoEl!, videoMediaStream!);
       },
+      clearOvalCanvas: (context) => {
```

Comment:
Can we run `clearOvalCanvas ` inside a `useEffect` to prevent having to pass the `canvas` element to the machine here?

Reply:
Certainly, updated the PR let me know what you think of the new pattern. Eventually we will need to update the other oval drawing functions to be like this but maybe can do more of that refactor in a separate pr